8.2.8 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

Commit:
fix(STN-315): word break issue (#567)

- Revert to default Decanter behavior as I think that's for the best

Commit:
feat: add Hi-Res support to WYSIWYG images styles (#568)

JIRA STN-259
https://sparkbox.atlassian.net/browse/STN-259

kaseybon committed:
feat: sprint 14 qa feedback (#571)

- Created hb-title utility
- Fixed button classes when used on a wrapper field
- Removed default external link icon, it can now be added with a class
- Fixed heading styles when alignment classes are added
- Updated hs-font-lead styles to be more distinctive

STN-319

su-sws-jira committed:
8.2.8

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr573-m96bw9pv6bpsvqg9zok9yq0ic01ivudv.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd

Visual Diffs:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd/visualdiffs

meranicosme committed:
feat(STN-321): update Humsci Colorful Sand color pairing with new colors (#572)

- adds compiled CSS as well

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr573-m96bw9pv6bpsvqg9zok9yq0ic01ivudv.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd

Visual Diffs:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd/visualdiffs

su-sws-jira committed:
Updated dependencies Apr 22 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr573-m96bw9pv6bpsvqg9zok9yq0ic01ivudv.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd

Visual Diffs:
- https://dashboard.tugboat.qa/5e9f62f6e59a48feeb9e61fd/visualdiffs

bryanbraun committed:
feat(STN-289): prevent menu flash during page-load (#574)

Our menus are expanded by default so users with JavaScript disabled can
navigate the site. This was contributing to a distracting flash of
expanded menus during the page load.

The solution was two-fold:

1. Preventing the flash of "no-js" styles, by removing the no-js class
   earlier (in the head of the document, instead of at the bottom).

2. Preventing the flash of expanded menus, by executing the "menu
   collapse JavaScript" immediately after the DOM nodes are added to the
   document.

Commit:
feat(STN-322) : More Link work (#578)

* feat: style the more link

* fix: use hs-button for infinate scroll button

* refactor: reset button.hs-more-link default styles, use animation vars in all transitions

* feat: add compiled CSS

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

Commit:
feat(STN-306): background color for Row Paragraph type (#577)

* feat(STN-306): adds new field for row paragraph type styles

- adds new config files for DB changes
- adds hook logic to add classes to DOM
- adds a couple preliminary styles for new row styles

* feat(STN-306): adds more well paragraph styles

* fix(STN-306): removes unecessary package entity_class_formatter

- I thought this could be a potential soln, but we ended up not needing
  it

* feat(STN-306): PR feedback:

- extends hb-well
- adds period to php comment
- adds compiled css

Commit:
feat(STN-324) : Detail Page Image Styles (#576)

* feat: create image styles utility class

* feat: implement design feedback

- adjust width property
- add margin for larger screen sizes

* feat: experiment with removing widths on the title and image for detail node pages

- remove utility classes that we have decided not to use
- clean up / remove comments
- refine styles for node detail pages

* feat: add comment about how image styles are handled

* feat: add complied CSS

kaseybon committed:
fix(STN-321): resolve IE JS errors, refactor secondary nav to use custom JS (#580)

pookmish committed:
HSD8-795 HSD8-789 Provision 2 new sites

pookmish committed:
HSD8-759 HSD8-761 Embedded media link changes and icons

pookmish committed:
HSD8-796 hide banner overlay text when in horizontal cards

pookmish committed:
HSD8-782 Add bulk field change actions from contrib module

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr573-yyuoe0f1fledfb5xafn6vgjdyfsnva5w.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ea8a11ec43236951cd84fb2

Visual Diffs:
- https://dashboard.tugboat.qa/5ea8a11ec43236951cd84fb2/visualdiffs

pookmish committed:
enabled react paragraphs

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr573-yyuoe0f1fledfb5xafn6vgjdyfsnva5w.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5ea8a11ec43236951cd84fb2

Visual Diffs:
- https://dashboard.tugboat.qa/5ea8a11ec43236951cd84fb2/visualdiffs

Commit:
STN-157 : Date Stacked Horizontal UI Pattern (#570)

* feat: create date stacked horizontal card UI pattern

JIRA STN-157
https://sparkbox.atlassian.net/browse/STN-157

* feat: apply date stacked horizontal card styles

* feat: adjust pattern to not render drupal output when field data is empty

* feat: implement feedback from design pairing

* fix: IE alignment bug

- replace the use of postion: initial with position: static
- IE doesn't support position: initial and was causing the content to stack vertically rather than display horizontally

* fix: linting error for empty line rule

* feat: add calendar icon and styles for the date and time label and field

* feat: add location and speaker icons and their styles

* feat: update class name to be more reflective of use case

* feat: scope icons to an hb-card__icon class

- this creates flexibility should we decide to implement icons in other templates that use the hb-card classes

* feat: add compiled CSS after feedback changes

su-sws-jira committed:
Updated dependencies Apr 29 2020

pookmish committed:
Provided config pages to allow easy default field values for images (#584)

pookmish committed:
Sparkbox Sprint 15

pookmish committed:
HSD8-784 Adjusted ui_patterns patch to check for empty values (#587)

pookmish committed:
Updated dependencies

pookmish committed:
re-exported config with updates

pookmish committed:
HSD8-797 Provision sparkbox-sandbox.stanford.edu
